feat: styles for typeahead dropdown

muhammad-ammar opened the pull request.

adamstankiewicz reviewed (commented):
These look to be the same styles as the ones added in https://github.com/edx/frontend-enterprise/pull/76?

If the styles already live in frontend-enterprise, we should be able to import them from frontend-enterprise into this MFE?

muhammad-ammar commented:
> These look to be the same styles as the ones added in [edx/frontend-enterprise#76](https://github.com/edx/frontend-enterprise/pull/76)?
> 
> If the styles already live in frontend-enterprise, we should be able to import them from frontend-enterprise into this MFE?

@adamstankiewicz yes, Initially I made style changes in `frontend-enterprise` repo but changes are not reflected on web page. Then I added the styles here and they are working. I am not sure who this whole flow. Can you guide what to do?

adamstankiewicz commented:
> > These look to be the same styles as the ones added in [edx/frontend-enterprise#76](https://github.com/edx/frontend-enterprise/pull/76)?
> > If the styles already live in frontend-enterprise, we should be able to import them from frontend-enterprise into this MFE?
> 
> @adamstankiewicz yes, Initially I made style changes in `frontend-enterprise` repo but changes are not reflected on web page. Then I added the styles here and they are working. I am not sure who this whole flow. Can you guide what to do?

@muhammad-ammar How were you importing the styles from `frontend-enterprise` into this MFE? For example, frontend-app-admin-portal imports styles from frontend-enterprise here: https://github.com/edx/frontend-app-admin-portal/blob/master/src/index.scss#L8

That import should bring in any styles imported by the SCSS file/partial `./course-search/styles/_index.scss` as seen here: https://github.com/edx/frontend-enterprise/blob/master/src/index.scss#L3

muhammad-ammar commented:
> > > These look to be the same styles as the ones added in [edx/frontend-enterprise#76](https://github.com/edx/frontend-enterprise/pull/76)?
> > > If the styles already live in frontend-enterprise, we should be able to import them from frontend-enterprise into this MFE?
> > 
> > 
> > @adamstankiewicz yes, Initially I made style changes in `frontend-enterprise` repo but changes are not reflected on web page. Then I added the styles here and they are working. I am not sure who this whole flow. Can you guide what to do?
> 
> @muhammad-ammar How were you importing the styles from `frontend-enterprise` into this MFE? For example, frontend-app-admin-portal imports styles from frontend-enterprise here: https://github.com/edx/frontend-app-admin-portal/blob/master/src/index.scss#L8
> 
> That import should bring in any styles imported by the SCSS file/partial `./course-search/styles/_index.scss` as seen here: https://github.com/edx/frontend-enterprise/blob/master/src/index.scss#L3

@adamstankiewicz I am not importing any styles from `frontend-enterprise` into this MFE because I have added new styles in existing `SCSS` files and my understanding is that new styles should work because existing `SCSS` should be already imported into this MFE.

adamstankiewicz commented:
@muhammad-ammar Right, but then we'll have duplicate styles defined in both frontend-app-learner-portal-enterprise and frontend-enterprise. These styles should only live in the frontend-enterprise library, and get imported into this MFE via its `src/index.scss` file.

Since the styles are for component(s) that live in frontend-enterprise, the styles should be co-located with the components in the library and have this MFE import the styles from frontend-enterprise rather than putting these styles in the MFE itself.

muhammad-ammar commented:
@raq929 @adamstankiewicz I am going to close this PR. As per my understanding styles added in `frontend-enterprise` should work without any issue.
